packages/extension-base/src/stores/Contacts.ts

```diff
@@ -26,7 +25,7 @@ const ContactsStore = {
   insert (contact?: Contact): void {
     const contacts = this.get();
     let newContacts = [];
-    const existContact = _.find(contacts, (item) => item.id === contact.id);
+    const existContact = contacts.find((item) => item.id === contact.id);
```

Comment:
Since you removed _.find, better to do `undefined` check on `contacts` here; otherwise undefined.find() causes error.